OpenTerm/Controller/TerminalViewController.swift

```diff
@@ -427,6 +427,7 @@ extension TerminalViewController: TerminalViewDelegate {
 
 		if command == "exit" {
 			if let parent = self.parent as? TerminalTabViewController {
+				terminalView.executor.closeSession()
```

Comment:
This isn't the only place a tab can be closed.
We probably want to override the `closeTab` function in `TerminalTabViewController`:
```swift
override func closeTab(_ tab: UIViewController) {
    super.closeTab(tab)

    if let terminalVC = tab as? TerminalViewController {
        terminalVC.terminalView.executor.closeSession()
    }

}
```

Once this is overridden, you can remove the `closeSession()` call in this `processCommand` function, since it already calls `closeTab`.